isisd/isis_cli.c

```diff
@@ -327,12 +327,8 @@ void cli_show_ip_isis_ipv6(struct vty *vty, struct lyd_node *dnode,
 /*
  * XPath: /frr-interface:lib/interface/frr-isisd:isis/bfd-monitoring
  */
-DEFPY(isis_bfd,
-      isis_bfd_cmd,
-      "[no] isis bfd",
-      NO_STR
-      PROTO_HELP
-      "Enable BFD support\n")
+DEFPY(isis_bfd, isis_bfd_cmd, "[no] isis bfd",
```

Comment:
Optional change: Was this on purpose? It looks a too different from other `DEFPY` declarations. I think we can ignore polychaeta's advice on this part.